kubernetes/samples/charts/apache-webtier/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
```

Comment:
Add Copyright?

Reply:
I will submit another change to add copyright.